File: Algorithms/Implementation/MinMaxSum.java
package Implementation;

import java.util.Scanner;

/**
 *	Algorithms ->  Implementation -> Mini-Max Sum
 *	Easy
 */
public class MinMaxSum {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
        long a = in.nextLong();
        long b = in.nextLong();
        long c = in.nextLong();
        long d = in.nextLong();
        long e = in.nextLong();
        in.close();
        long[] array = {
        		a, b, c, d, e
        };
        long min = a + b + c + d + e;
        long max = 0;
        for(int i = 0; i < array.length; i++) {
        	long sum = 0;
        	for(int j = 0; j < array.length; j++) {
        		if(i != j) sum += array[j];
        	}
        	if(sum > max) max = sum;
        	if(sum < min) min = sum;
        }
        System.out.println(min + " " + max);
	}

}

File: Algorithms/Implementation/MinMaxSum.c
#include <math.h>
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <assert.h>
#include <limits.h>
#include <stdbool.h>

/**
 *	Algorithms -> Warmup -> A Very Big Sum
 *	Easy
 */
int main(){
	long long a, b, c, d, e;

	scanf("%lld %lld %lld %lld %lld",&a,&b,&c,&d,&e);

	long long array[] = {a, b, c, d, e};

	long long min = a + b + c + d + e;
	long long max = 0;

	for(int i = 0; i < 5; i++) {
		long long sum = 0;
		for(int j = 0; j < 5; j++) {
			if(i != j) sum += array[j];
		}
		if(sum > max) max = sum;
		if(sum < min) min = sum;
	}

	printf("%lld %lld", min, max);

    return 0;
}
